### ISSUE TITLE
Some masterworks have more than one stat

### ISSUE THREAD

**Issue by justrealmilk** [DestinyItemManager/DIM#5364]
Original title: Adjust [inventory/store/masterwork.ts](https://github.com/DestinyItemManager/DIM/blob/master/src/app/inventory/store/masterwork.ts#L87) to return array of stats for rare instances where masterwork has more than one stat.

For instance, Coldheart.

![image](https://user-images.githubusercontent.com/156681/85320610-ae7bc680-b506-11ea-9e34-d2ee30791137.png)

i.e.
```js
stats: socket.plug.definition.investmentStats.map((stat) => ({
  hash: stat.statTypeHash,
  value: socket.plug.stats[stat.statTypeHash],
})),
```

![image](https://user-images.githubusercontent.com/156681/85320787-fc90ca00-b506-11ea-8b20-9c0bba9d443d.png)

ta da

**Comment by delphiactual** [comment]
I wonder if #5333 is related

**Comment by justrealmilk** [comment]
Actually, as it turns out, a fair few exotic catalysts provide more than one investment stat bonus

**Comment by duckmanBR** [comment]
When I coded this 2 years ago they probably didn't...

**Comment by sundevour** [comment]
closed by #5429 ?
